authors/settings/prod.py

```diff
@@ -6,6 +6,11 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
+ALLOWED_HOSTS = ['ah-legion.herokuapp.com']
```

Comment:
@BrianSerem, make allowed hosts '*' for all connections to be allowed, or pick allowed hosts from env variables so it can be configurable.

Reply:
> @BrianSerem, make allowed hosts '*' for all connections to be allowed, or pick allowed hosts from env variables so it can be configurable.

I picked allowed hosts from the env file, thanks for catching that